libraries/testkit/syncgateway.py

```diff
@@ -62,6 +63,15 @@ def start(self, config):
             playbook_vars["autoimport"] = '"import_docs": "continuous",'
             playbook_vars["xattrs"] = '"enable_shared_bucket_access": true,'
 
+        if is_cbs_ssl_enabled(self.cluster_config) and get_sg_version(self.cluster_config) >= "1.5.0":
```

Comment:
This is a string comparison. compare_versions(version_one, version_two): in keywords.utils might be a better fit.